packages/did/src/Did.utils.ts

```diff
@@ -107,6 +106,14 @@ export function parse(did: Did | DidUrl): IDidParsingResult {
       ? parseInt(versionString, 10)
       : LIGHT_DID_LATEST_VERSION
     const queryParameters = exportQueryParamsFromDidUrl(did as DidUrl)
+    // checking if encodedDetails can be decoded
+    if (encodedDetails) {
+      try {
```

Comment:
The library has a validation function. We should be able to use that. @rflechtner What do you think?

```
base58Validate(encodedDetails, true)
```